### DISCUSSION TITLE
Release/Build Numbers

### DISCUSSION THREAD

**Discussion by eelliston** [qetza/replacetokens-task#57] [Q&A]
I have been using this task for quite some time, it has been great. I need to make a slight change where pass in the release/build number (from Azure DevOps system variables) so it is replaced in my code.  We have a version.json file that we are going to use to help us with some cache busting.  

I can't figure out how to pass in ADO system variables. I am only able to pass in library variables or task variables.  

I don't really use YAML for this, my tasks were build via that ADO UI designer.  :)... 

Any direction would be awesome!

Thanks!

**Comment by qetza** [comment]
Hi @eelliston,
You should be able to replace tokens with ADO predefined variables by using the variable name. For example the token `#{Build.BuildNumber}#` will be replace by the build number.
